src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+

```

Comment:
Avoid starting the method implementation with an empty line. Remove this redundant empty line as per the checklist guidelines.